storage/innobase/trx/trx0undo.cc

```diff
@@ -1015,6 +1015,20 @@ trx_undo_mem_create_at_db_start(trx_rseg_t *rseg, ulint id, uint32_t page_no)
 	case TRX_UNDO_ACTIVE:
 	case TRX_UNDO_PREPARED:
 		if (UNIV_LIKELY(type != 1)) {
+			/* Previosly committed undo log can have trx_no greater
+			than trx_id of the current undo log, because the trx_id
+			of the current undo log can be assigned before the
+			previous undo log is committed. */
```

Comment:
Typo: "previously". I would also slightly clarify the terminology and refer to committing transactions rather than undo logs (which represent the changes that are made by transactions):
```c
			/* The undo log of a previously committed
			transaction that was logged in this page may
			have trx_no greater than the trx_id of the
			current undo log, because the trx_id of the
			current transaction can be assigned before the
			previous transaction is committed. */
```